core/common/src/main/java/alluxio/AbstractClient.java

```diff
@@ -173,6 +179,16 @@ public synchronized void connect() throws IOException, ConnectionFailedException
         afterConnect();
         checkVersion(getClient(), getServiceVersion());
         return;
+      } catch (IOException e) {
+        if (FRAME_SIZE_EXCEPTION_PATTERN.matcher(e.toString()).find()) {
```

Comment:
Actually `e.toString()` is usually short (typically one line), callstack info is not in it